src/Dropdown/Dropdown.mdx

```diff
@@ -73,6 +74,24 @@ Easily display contextual overlays using custom trigger elements. Dropdown's pos
 
               <Dropdown.Footer>Footer</Dropdown.Footer>
             </Dropdown>
+
+            <div>
+              <Dropdown
+                width={250}
+                placement="top"
+                trigger={<Icon name="information-outline" />}
```

Comment:
In order to make icon buttons accessible, we can wrap the icon in a div with role=Button and tabIndex=0. https://louiseclark.tech/tech-blog/accessibility-journey-making-buttons-accessible